app/controllers/items_controller.rb

```diff
@@ -4,11 +4,40 @@ class ItemsController < ApplicationController
   include ItemsHelper
   include DorObjectHelper
   include ModsDisplay::ControllerExtension
-  before_filter :create_obj, :except => [:register, :open_bulk, :purge_object]
-  before_filter :forbid_modify, :only => [:add_collection, :set_collection, :remove_collection, :update_rights, :set_content_type, :tags, :tags_bulk, :source_id, :delete_file, :close_version, :open_version, :resource, :add_file, :replace_file, :update_attributes, :update_resource, :mods, :datastream_update ]
-  before_filter :forbid_view,   :only => [:get_preserved_file, :get_file]
-  before_filter :enforce_versioning, :only => [:add_collection, :set_collection, :remove_collection, :update_rights, :tags, :source_id, :set_source_id, :set_content_type, :set_rights]
-  after_filter  :save_and_reindex,   :only => [:add_collection, :set_collection, :remove_collection, :open_version, :close_version, :tags, :tags_bulk, :source_id, :set_rights, :set_content_type, :apply_apo_defaults]
+  before_filter :create_obj, :except => [
```

Comment:
Appease the hound.ci or the PR doesn't get the green light.  (Discussed in slack and can be discussed further in person.  Understand the pain of reviewing something like this where it's difficult to focus on the main thing.  Will learn from this experience and raise the learning experience in the DLSS best-practices forum.)